app/views/publish/courses/index.html.erb

```diff
@@ -3,6 +3,8 @@
 
 <h1 class="govuk-heading-l">Courses</h1>
 
+<% if (current_recruitment_cycle?(@provider) || @provider.study_sites.any? || !FeatureService.enabled?(:study_sites)) %>
```

Comment:
There will be a few more conditions where we need to hide the "Add course" button. I will probably make a component then to clean up the view. 

See: https://becoming-a-teacher.design-history.education.gov.uk/publish-teacher-training-courses/hiding-the-add-course-button-when-organisation-details-are-missing/